test/integration/services/RabbitMQService.test.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const AMQPStats = require('amqp-stats');
+const should = require('should');
+
+/**
+ * RabbitMQ connection.
+ */
+describe('RabbitMQ', () => {
+  /**
+   * Helper: get default client.
+   */
+  function getManagementClient() {
+    return new AMQPStats({
+      username: sails.config.amqp.management.username,
+      password: sails.config.amqp.management.password,
+      hostname: sails.config.amqp.management.hostname,
+      protocol: sails.config.amqp.management.protocol,
```

Comment:
I'm not clear why you would need / want a connection to a RabbitMQ Management client in the "Dirty" implementation? It's not needed in terms of doing basic sending of messages.  Perhaps you need it for a health check?